### ISSUE TITLE
Unknown error setting up integration

### ISSUE THREAD

**Issue by JesperThorup** [MTrab/landroid_cloud#357]
### Describe the issue

When running the initial setup, after entering my mail and password, pressing the send button, I just get an unknown error.

I don't have mush more to go with...

### What version of Home Assistant Core has the issue?

2023.4.6

### What version of the Landroid Cloud integration do you have installed

v3.0.0b5

### What type of installation are you running?

Home Assistant OS

### Which make and model is the mower used for this integration?

Worx WR147E.1

### Diagnostics information (NOT log entries!)

Cannot provinde any diagnostic data as I cannot get past the initial setup..

### Relevant log entries

```txt
Logger: custom_components.landroid_cloud.config_flow
Source: custom_components/landroid_cloud/utils/logger.py:126
Integration: Landroid Cloud (documentation, issues)
First occurred: 08.57.59 (2 occurrences)
Last logged: 09.01.41

(Config) Unexpected exception:
```

**Comment by MTrab** [comment]
Please go find the relevant log entries in your log file - there might be more useful information in there.

**Comment by JesperThorup** [comment]
> Please go find the relevant log entries in your log file - there might be more useful information in there.

Where do I find the log file? Because there are nothing else under system -> logs that points to the Landroid intergration..

**Comment by MTrab** [comment]
It's the file called home-assistant.log

**Comment by JesperThorup** [comment]
@MTrab The only line regarding Landroid is:
2023-04-27 08:49:11.974 ERROR (MainThread) [custom_components.landroid_cloud.config_flow] (Config) Unexpected exception:

**Comment by MTrab** [comment]
And nothing else at all?
In that case, enable debug logging and see what it tells you then

**Comment by JesperThorup** [comment]
@MTrab Yes some informational and warnings for other integrations...

How do I enable debug logging, when I cant do it directly on the integration?

**Comment by MTrab** [comment]
Add this to your configuration.yaml file if you cannot simply enable debugging from the integrations page

```
logger:
  default: warning
  logs:
    custom_components.landroid_cloud: debug
```

**Comment by JesperThorup** [comment]
Then I get this:

2023-04-27 10:13:53.792 DEBUG (MainThread) [custom_components.landroid_cloud.config_flow] (Config) data: {'email': 'xxxxxx@xxxx.dk', 'password': 'xxxxxxxxx', 'type': 'Worx'}
2023-04-27 10:13:53.792 DEBUG (MainThread) [custom_components.landroid_cloud.config_flow] (Config) This works
2023-04-27 10:13:54.000 ERROR (MainThread) [custom_components.landroid_cloud.config_flow] (Config) Unexpected exception:

**Comment by JesperThorup** [comment]
Damn.... It was a typo in the password... 

Sorry...

**Comment by MTrab** [comment]
Yeah - I need to get that check fixed so it reports that back correctly

**Comment by MTrab** [comment]
Think I'll let this issue stay open just to mind myself to get that check fixed ;)

**Comment by phsluis** [comment]
I am unable to setup the integration, the following errors appear in the log.

**Comment by MTrab** [comment]
Not same error as this - your error is because you are not on the beta version of the integration, as mentioned on the frontpage

**Comment by phsluis** [comment]
Thank you, I understand. I will try with the beta version.

**Comment by phsluis** [comment]
Quick update, with the beta version it works.

**Comment by garmcqui** [comment]
> 

Can I ask how we get on the beta version please? Thanks

**Comment by roofburner** [comment]
If I’m not mistaken :

Home Assistant -> HACS -> Integrations -> Landroid Cloud -> Three dots in the upper right corner -> Redownload -> Show beta versions -> v3.0.0b5 -> Download -> restart Home Assistant

________________________________
Van: garmcqui ***@***.***>
Verzonden: zaterdag 6 mei 2023 18:16
Aan: MTrab/landroid_cloud ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: Re: [MTrab/landroid_cloud] Unknown error setting up integration - wrong password used (Issue #357)


Can I ask how we get on the beta version please? Thanks
